src/Handler/ArchiveHandler.php

```diff
@@ -2,9 +2,11 @@
 
 namespace LastCall\DownloadsPlugin\Handler;
 
+use Clue\React\Block;
```

Comment:
Argh this is an unused import from a library I used to wait for a promise to resolve until I realized Composer had a helper method.